packages/@sanity/form-builder/src/inputs/arrays/common/ArrayFunctions.tsx

```diff
@@ -38,6 +38,8 @@ export default function ArrayFunctions<MemberType>(
     insertItem(type.of[0])
   }, [type, insertItem])
 
+  const popoverProps = useMemo(() => ({constrainSize: true, portal: true}), [])
```

Comment:
Any reason not to make this a constant?

Reply:
That feels like moving it too far away from the context. And if for some reason the value should be dynamic at a later stage, it’s a shorter path to that now, e.g.:

```tsx
function ArrayFunctions (props: ArrayFunctionProps) {
  const {portal: portalProp = true} = props
  
  const popoverProps = useMemo(() => ({constrainSize: true, portal: portalProp}), [portalProp])
```